### PR TITLE
Trigger rubocop only if ruby files has been changed

### PR DESCRIPTION
**Pull request by szachovy**
Request from the discussion: https://suse.slack.com/archives/C02D12TNYLS/p1729753176627919

### CODE REVIEW

File: jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop [hunk 1]
```diff
@@ -51,12 +56,17 @@ pipeline {
             }
         }
         stage('Run tests against PR') {
+            when {
+                anyOf {
+                    changeset "**/*.rb"
```
Comment (srbarrios) [inline jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop:61]: That will not take into account Rakefile

Comment (szachovy) [reply jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop:64]: In GH actions we have it like that: https://github.com/SUSE/spacewalk/blob/6fbfe9b94e87a0c1b0d025c3a266fc84c396b3ba/.github/workflows/rubocop.yml#L5

I followed this convention, in the recent commit also for `yml` files. I have no problem to add `Gemfile` and `Rakefile` if needed.

Comment (srbarrios) [reply jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop:64]: Good catch! I think we should include both Gemfile and Rakefile in both GH Actions and Jenkinsfile. As few days ago, we had an issue in Rakefile if I recall correctly. Would you mind to take care of both changes please?

Comment (szachovy) [reply jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop:66]: Done

**Review by maximenoel8 — APPROVED** [review]

**Review by ktsamis — APPROVED** [review]

**Review by srbarrios — COMMENTED** [review]
Thank you!

**Pull request merged**